src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ManagedInstanceKeyVaultKeyTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ScenarioTest.SqlTests;
+using Microsoft.Azure.ServiceManagemenet.Common.Models;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.Sql.Test.ScenarioTests
+{
+    public class ManagedInstanceKeyVaultKeyTests : SqlTestsBase
+    {
+        public ManagedInstanceKeyVaultKeyTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact(Skip = "Skip due to long setup time for managed instance")]
```

Comment:
It looks like all the tests are skipped now, we need at least one test running for this.

Reply:
Also, instead of having [Fact(Skip)], please just comment out [Fact] on the tests that you don't want to run on CI.  We often go through to find skipped tests to fix them, so easier to just comment out the [Fact] label to indicate that this isn't a test for CI.

Reply:
I was told before to skip the tests that do not have automated setup? Is is okay to mark the test without automated setup as [Fact] ?

Reply:
Yes, that's fine.  We just need detailed instructions about how to set up the test for recording if we need to rerecord.

Reply:
Addressed